handlers/deploy.go

```diff
@@ -190,8 +190,11 @@ func buildEnv(envProcess string, envVars map[string]string) []string {
 // BuildEncodedAuthConfig for private registry
 func BuildEncodedAuthConfig(basicAuthB64 string, dockerImage string) (string, error) {
 	// extract registry server address
+	if len(strings.Split(dockerImage, "/")) < 3 {
```

Comment:
I think we can document this better.. when we read it again in 6 months it will look like magic

Reply:
Good point!